.travis.yml

```diff
@@ -6,13 +6,14 @@ env:
   - distro: centos6
   - distro: ubuntu1604
   - distro: ubuntu1404
-  - distro: ubuntu1204
-  - distro: fedora24
+  - distro: fedora25
+  - distro: fedora26
```

Comment:
Could you use a separated PR for this? I mean is better to have one PR for bug fixes and second one for feature request

Reply:
That would be cleaner. I will get one for changing the include syntax which removes the older distros and add another for the new distros.